### PR TITLE
Follow-up edits to Samples Operator documentation

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/13625

This will include formatting changes to match the outlined template. 

Also addressing findings from https://bugzilla.redhat.com/show_bug.cgi?id=1680519.

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -164,6 +164,8 @@ Name: Images
 Dir: openshift_images
 Distros: openshift-*
 Topics:
+- Name: Configuring the Samples Operator
+  File: configuring-samples-operator
```
Comment (xiuwang) [inline _topic_map.yml]: "Create images" and  "Managing image sterams" topics need move up too

**Comment by bmcelvee** [comment]
@gabemontero here's the follow-up edits PR. I'm also incorporating some changed request in the phase 1 docs review.

File: modules/samples-operator-configuration.adoc [hunk 2]
```diff
@@ -36,14 +34,15 @@ removal has started.
...
-Creation or update of RHEL content is not be gated if the `SamplesRegistry` has
-been overridden.
+Creation or update of RHEL content is not be gated if the `Samples Registry` is
+overridden to a value other than the empty string or registry.redhat.io.
 ====
 +
-* *Architecture:* Place holder to choose an architecture type. Currently only x86
+* *Architectures:* Place holder to choose an architecture type. Currently only x86
 is supported.
 * *Skipped Image streams:* Image streams that are in the operator’s
```
Comment (xiuwang) [inline modules/samples-operator-configuration.adoc]: “Skipped Image streams‘ should update to 'Skipped Imagestreams'

File: modules/samples-operator-configuration.adoc [hunk 3]
```diff
@@ -9,21 +9,19 @@
 
 The samples resource offers the following configuration fields:
 
-* *ManagementState:*
+* *Management State:*
 ** *Managed.* The operator updates the samples as the configuration dictates.
-** *Unmanaged.* The operator ignores updates to the samples resource object and
-any image streams or templates in the OpenShift namespace.
+** *Unmanaged.* The operator ignores updates to its configuration resource object
+and any image streams or templates in the OpenShift namespace.
```
Comment (xiuwang) [inline modules/samples-operator-configuration.adoc]: All 'image streams' phrases in this doc https://docs.openshift.com/container-platform/4.0/openshift_images/configuring-samples-operator.html#samples-operator-configuration-configuring-samples-operator should be updated to 'imagestreams'

File: modules/samples-operator-configuration.adoc [hunk 4]
```diff
@@ -9,21 +9,19 @@
 
 The samples resource offers the following configuration fields:
 
-* *ManagementState:*
+* *Management State:*
 ** *Managed.* The operator updates the samples as the configuration dictates.
-** *Unmanaged.* The operator ignores updates to the samples resource object and
-any image streams or templates in the OpenShift namespace.
+** *Unmanaged.* The operator ignores updates to its configuration resource object
```
Comment (xiuwang) [inline modules/samples-operator-configuration.adoc]: Better to keep ’configuration resource‘ with  'samples resource' in line

File: modules/samples-operator-configuration.adoc [hunk 5]
```diff
@@ -36,14 +34,15 @@ removal has started.
...
 Creation or update of RHEL content does not commence if the secret for pull
-access is not in place when using the default, registry.redhat.io, when
-`SamplesRegistry` is not explicitly set.
+access is not in place when either `Samples Registry` is not explicitly set (i.e.,
+the empty string), or when it is set to registry.redhat.io. In both cases, image
+imports will work off of registry.redhat.io, which requires credentials.
 
-Creation or update of RHEL content is not be gated if the `SamplesRegistry` has
-been overridden.
+Creation or update of RHEL content is not be gated if the `Samples Registry` is
```
Comment (gabemontero) [inline modules/samples-operator-configuration.adoc]: s/ is not be gated / is not gated by the existence of the pull secret

File: modules/samples-operator-overview.adoc [hunk 6]
```diff
@@ -7,37 +7,37 @@
...
-* Operator managed templates
-* Operator generated configuration resources
-* Cluster status resources
-* The `samples-registry-credentials` secret
+definitions for the associated OpenShift release. When each sample is created or
+updated, the Samples Operator includes an annotation that denotes the
+version of {product-title}. The operator uses this annotation to ensure
+that each sample matches the release version. Samples outside of its inventory
+are ignored, as are skipped samples. Modifications to any samples that are
+managed by the operator will be reverted automatically. The Jenkins images are
```
Comment (gabemontero) [inline modules/samples-operator-overview.adoc]: managed by the operator -> managed by the operator, where that version annotation is modified or deleted,

File: modules/samples-operator-overview.adoc [hunk 7]
```diff
@@ -7,37 +7,37 @@
...
+image streams directly.
+
+The Samples Operator configuration resource includes a finalizer which cleans up
+the following upon deletion:
+
+* Operator managed imagestreams.
+* Operator managed templates.
+* Operator generated configuration resources.
+* Cluster status resources.
+* The `samples-registry-credentials` secret.
```
Comment (gabemontero) [inline modules/samples-operator-overview.adoc]: we are planning to no longer delete the secret with https://github.com/openshift/cluster-samples-operator/pull/125 ... please track that PR, and assuming that goes through, remove this line please

**Review by gabemontero — CHANGES_REQUESTED** [review]
Some minor changes @bmcelvee

File: _topic_map.yml [hunk 8]
```diff
@@ -164,6 +164,8 @@ Name: Images
...
 - Name: Understanding containers, images, and image streams
```
Comment (xiuwang) [inline _topic_map.yml]: s/image stremas/imagestreams

File: modules/samples-operator-configuration.adoc [hunk 9]
```diff
@@ -36,43 +35,44 @@ removal has started.
...
+Creation or update of RHEL content is not gated by the existence of the pull
+secret if the `Samples Registry` is overridden to a value other than the empty
+string or registry.redhat.io.
 ====
 +
-* *Architecture:* Place holder to choose an architecture type. Currently only x86
+* *Architectures:* Place holder to choose an architecture type. Currently only x86
 is supported.
-* *Skipped Image streams:* Image streams that are in the operator’s
+* *Skipped imagestreams:* Imagestreams that are in the Samples Operator’s
```
Comment (xiuwang) [inline modules/samples-operator-configuration.adoc]: s/Skipped imagestreams/Skipped Imagestreams

**Comment by xiuwang** [comment]
LGTM, thanks

**Pull request merged**